tests/measure/editors/test_execution_editor.py

```diff
@@ -35,6 +33,11 @@
 def task():
     """Create a basic task hierarchy for testing.
 
+    Root
+      - SimpleTask: parallel=False, wait=False
+      - ComplexTask: parallel=False, wait=True
+        - SimpleTask: parallel=True, wait= False
```

Comment:
Aaah ok got it, didn't read it well. 4 space would be clearer but that's not very important